internal/levee_test.go

```diff
@@ -29,10 +30,18 @@ var patterns = []string{
 	"example.com/tests/sinks",
 }
 
+var debugging bool
+
 func TestLevee(t *testing.T) {
 	dir := analysistest.TestData()
 	if err := Analyzer.Flags.Set("config", dir+"/test-config.json"); err != nil {
 		t.Error(err)
 	}
+	if debugging {
```

Comment:
Is it possible to turn this into a flag?
Related to the above comment: when the debug flag is set, could we include the `debug.Analyzer` as a prerequisite for the regular analyzer.

Reply:
Do you mean a test flag? We don't want our users to be able to run this code, right?

Reply:
I meant `Analyzer.Flags.Set`